### PR TITLE
update files for 0.7.0 release

### PR DESCRIPTION
**Pull request by elmiko**
updating for the release, once this merges we can close the tracking issue #104

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@
 # between patch versions.
 #
 # update this file only when a new major or minor version is released
-apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
+apiVersion: clusterctl.cluster.x-k8s.io/v1alpha4
```
Comment (killianmuldoon) [inline metadata.yaml:6]: clusterctl is still at `v1alpha3` - I don't think this should be updated.

Comment (killianmuldoon) [reply metadata.yaml:6]: c/f 
https://github.com/kubernetes-sigs/cluster-api/blob/de4b6521117ad5a473548c3388112d8af8209566/metadata.yaml#L6

Comment (elmiko) [reply metadata.yaml:6]: lol, i thought i was being overly smart here. thanks!

**Comment by elmiko** [comment]
updated with suggestion

**Pull request merged**